doc/getting_started/setup_fpga.md

```diff
@@ -148,15 +148,15 @@ To ensure that you have sufficient access permissions, set up the udev rules as
 
 ## Flash the bitstream onto the FPGA
 
-To flash the bitstream onto the FPGA using the `cw310_loader` tool, use the following command:
+To flash the bitstream onto the FPGA using `opentitantool`, use the following command:
 
 ```console
 cd $REPO_TOP
 
 # If you downloaded the bitstream:
-./util/fpga/cw310_loader.py --bitstream build-bin/hw/top_earlgrey/lowrisc_systems_chip_earlgrey_cw310_0.1.bit
+bazel run //sw/host/opentitantool load-bitstream build-bin/hw/top_earlgrey/lowrisc_systems_chip_earlgrey_cw310_0.1.bit
 # If you built the bitstream yourself:
-./util/fpga/cw310_loader.py --bitstream build/lowrisc_systems_chip_earlgrey_cw310_0.1/synth-vivado/lowrisc_systems_chip_earlgrey_cw310_0.1.bit
+bazel run //sw/host/opentitantool load-bitstream build/lowrisc_systems_chip_earlgrey_cw310_0.1/synth-vivado/lowrisc_systems_chip_earlgrey_cw310_0.1.bit
```

Comment:
Should the command point to the bitstream generated by bazel? `bazel-out/...` or `bazel-bin/...`

Reply:
You can may be able to use something like ```$($REPO/ci/scripts/target-location.sh //hw/bitstream/vivado:fpga_cw310_test_rom)``` to get a path for the bitstream from bazel. I'm working on getting it building so I can figure out the appropriate label

Reply:
Thanks @drewmacrae! This gave me `bazel-out/k8-fastbuild/bin/hw/bitstream/vivado/build.fpga_cw310/synth-vivado/lowrisc_systems_chip_earlgrey_cw310_0.1.bit` which looks correct to me.